core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -57,66 +58,70 @@ var LearnerDashboardPage = function() {
     await waitFor.elementToBeClickable(
```

Comment:
The action click function waits on elements to be clickable internally before clicking on it. So, here and below, please drop the "waitFor.elementToBeClickable" before "action.click" since it is redundant.